api-reference/10 UI Widgets/dxLoadPanel/3 Methods/toggle(showing).md

```diff
@@ -0,0 +1,4 @@
+You can use the this method instead of the **show()** and **hide()** methods. In this case, pass **true** or **false** as the method parameter to show or hide the widget.
```

Comment:
```suggestion
You can use this method instead of the **show()** and **hide()** methods. In this case, pass **true** or **false** as the method parameter to show or hide the widget.
```